code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -714,3 +714,23 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 		return // no material modding
 	else
 		..()
+
+/obj/item/clothing/under/abductor
+	desc = "The most advanced form of jumpsuit known to reality, looks uncomfortable. This one is in a purple color."
+	name = "alien jumpsuit"
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "abductor-suit"
+	item_state = "bl_suit"
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 10, bio = 10, rad = 0, fire = 0, acid = 0)
+	can_adjust = 0
+	species_restricted = list("exclude","Vox Outcast")
+
+/obj/item/clothing/under/abductor/blue
+	desc = "The most advanced form of jumpsuit known to reality, looks uncomfortable. This one is in a turquoise color."
+	name = "alien jumpsuit"
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "abductor-suitblue"
+	item_state = "bl_suit"
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 10, bio = 10, rad = 0, fire = 0, acid = 0)
+	can_adjust = 0
+	species_restricted = list("exclude","Vox Outcast")
```

Comment:
everything that isn't changed on this jumpsuit is redundant. `.../abductor/blue` inherits everything from `.../abductor` unless explicitly overwritten

Reply:
Done, @FlattestGuitar , removed the redundancies.